extensions/wikia/UserProfilePageV3/css/UserProfilePage_modal.scss

```diff
@@ -15,13 +16,22 @@
 
 	.input-group {
 		float: none;
-		position: relative;
 
 		input[type="text"],
 		.favorite-wikis {
 			@include box-sizing-border-box;
 			width: 100%;
 		}
+
+		.tweet-at, .fb-start {
+			float: left;
+			margin-top: 10px;
+			width: 110px;
+		}
+
+		input[name="fbPage"], input[name="twitter"] {
```

Comment:
Oops, looks like you're correct. I probably had an old branch. I'm seeing the new forms that you mention. That was my mistake.